instrumentation/kubernetes-client-7.0/src/main/java/io/opentelemetry/auto/instrumentation/kubernetesclient/KubernetesClientTracer.java

```diff
@@ -53,4 +52,32 @@ protected String requestHeader(Request request, String name) {
   protected String responseHeader(Response response, String name) {
     return response.header(name);
   }
+
+  @Override
+  protected Setter<Request> getSetter() {
+    return null;
+  }
+
+  @Override
+  protected String getInstrumentationName() {
+    return "io.opentelemetry.auto.kubernetes-client-7.0";
+  }
+
+  @Override
+  protected Span onRequest(Span span, Request request) {
+    return super.onRequest(span, request);
+  }
```

Comment:
```suggestion
```

Reply:
This is called by TracingInterceptor.java line 38.